types/office-js-preview/index.d.ts

```diff
@@ -521,6 +525,15 @@ declare namespace Office {
         */
         value: T;
     }
+    /**
+     * Used to associate an action.
```

Comment:
I'm a little confused by this. Is this Actions object only for one action?

Reply:
No, it can be used to provide a mapping between a name with an action that can be used for multiple actions.

Reply:
Made changes proposed by @Rick-Kirkham .